pyLRT/RadTran.py

```diff
@@ -112,7 +106,108 @@ def run(self, verbose=False, print_input=False, print_output=False, regrid=True,
             return (np.genfromtxt(io.StringIO(process.stdout)),
                     _read_verbose(io.StringIO(process.stderr), regrid=regrid))
         return np.genfromtxt(io.StringIO(process.stdout))
-
+    
+    def add_cloud(
+        self, type="liquid", height=None, base_height=None, thickness=None, 
+        lwc=None, iwc=None, re=None, od=None
+    ):
+        '''Add a cloud layer'''
+        if height is None:
+            raise ValueError("height must be provided")
+        if base_height is None:
+            if thickness is None:
+                raise ValueError("thickness must be provided")
+            else:
+                base_height = height-thickness
+        if re is None:
+            raise ValueError("re must be provided")
+        if lwc is not None:
+            type = "liquid"
+        if iwc is not None:
+            type = "ice"
+        if od is not None:
+            # Convert OD to CWP: cwp in g/m2, re in microns
+            cwp = 2/3 * od * re * 1e-3
+            if type=="liquid":
+                lwc = cwp
```

Comment:
If I am reading this right, cwp is the cloud water path for a liquid cloud with a droplet size that is independent of height. I think in that case the entry into the cloud table should have re at both the bottom and the top? 

In that case, is lwc/iwc equal to cwp/2 (assuming lwc increases linearly with height)?

Reply:
I think I might have done this wrong in the example too....